Cargo.toml

```diff
@@ -23,7 +23,7 @@ doctest    = false
 path       = "src/lib.rs"
 
 [dependencies]
-test-case-macros = { version = "2.2.1", path = "crates/test-case-macros", default-features = false }
+test-case-macros = { version = "2.2.2", path = "crates/test-case-macros", default-features = false }
```

Comment:
Seemed this bump was just missing, but lmk if I am missing something

Reply:
That's odd, coz on crates.io we definitely have correct one released